layouts/partials/head.html

```diff
@@ -69,7 +69,22 @@
     });
   </script>
   {{ end }}
-
+  {{ "<!-- ENTERING LDJSON for events -->" | safeHTML }}
+  {{ if .IsPage }}
+  {{ if eq .Type "events" }}
+  <script type="application/ld+json">
+  {
+      "@context": "https://schema.org",
+      "@type": "Event",
+      "name": "{{ .Params.Title }}",
+      "startDate": "{{ .Params.date.Format "2006-01-02T15:04:05JST" }}",
+      "url": "{{ .Params.youtubeLink }}",
+      "image": "{{ .Params.image | absURL }}",

```

Comment:
```suggestion
      "image": "{{ .Params.image | absURL }}"
```

JSON with a comma before `}` is not valid, the validator from google complained about it.

Also, I tested this with the InnerSource Summit 2025, and the output was:
```
  <script type="application/ld+json">
  {
      "@context": "https://schema.org",
      "@type": "Event",
      "name": "InnerSource Summit 2025",
      "startDate": "2025-11-13T00:00:00JST",
      "url": "",
      "image": "http:\/\/localhost:1313\/images\/events\/Summit2025.png",
  }
  </script>
```

The validator complained about missing location and that the URL image field is invalid. For the URL image, the problem seems to be just that the test app from google can't reach the URL because it is local host, when I manually changed to `https:\/\/innersourcecommons.org\/images\/events\/Summit2025.png`, that error was gone.

Reply:
@dellagustin-sap yeah that sounds about right. With the location we don't necessarily have location for events so we might as well just let it complain as long as it doesn't forfeit the functionality

Reply:
@andrea-kyurchiev , yeah, even if google does not support virtual locations, the definition of Event does. Location could be of the type `VirtualLocation`. Maybe other platforms will support it. It would be interesting to have an overview of where this is supported, I'll share if I find something.

Once the tickets sale for the summit is live, I'll check if the platform we are using is already providing such metadata, I think it is (based on https://developers.google.com/search/docs/appearance/structured-data/event, _Increased chances of discovery and conversion: People have a new way to interact with your event posting and click through to your site. Learn how [Eventbrite saw a 100% increase](https://developers.google.com/search/case-studies/eventbrite-case-study) in the typical year-over-year growth of traffic from Google Search._)

I don't know if google or other search engines are still supporting this, as I did some tests in google and Bing and could not get any results with special event cards or something...

References:
* https://schema.org/Event
* https://schema.org/VirtualLocation